File: src/main/java/com/PrzeBarCore/Laboratorymanagementsystem/dto/events/MedicalOrderStatusChangedEvent.java
package com.PrzeBarCore.Laboratorymanagementsystem.dto.events;

import com.PrzeBarCore.Laboratorymanagementsystem.global.OrderStatus;

import java.time.LocalDateTime;

public record MedicalOrderStatusChangedEvent(Long orderId, OrderStatus oldStatus, OrderStatus newStatus, LocalDateTime changedAt) {
}

File: order-monitoring-worker/src/main/kotlin/com/PrzeBarCore/ordermonitoringworker/dto/MedicalOrderStatusChangedEvent.kt
package com.PrzeBarCore.ordermonitoringworker.dto

import com.PrzeBarCore.ordermonitoringworker.global.enums.OrderStatus
import java.time.LocalDateTime

data class MedicalOrderStatusChangedEvent(val orderId: Long,
                                          val oldStatus: OrderStatus,
                                          val newStatus: OrderStatus,
                                          val changedAt: LocalDateTime
)
